lib/extensions.js

```diff
@@ -69,7 +69,8 @@ function getHumanNodeVersion(abi) {
     case 47: return 'Node.js 5.x';
     case 48: return 'Node.js 6.x';
     case 51: return 'Node.js 7.x';
-    case 54: return 'Node.js 8.x';
+    case 55: return 'Node.js 8.x';
```

Comment:
True. I did them as discrete commits so I didn't think about it. I want to update this handling so that it only shows as a _helpful_ warning rather than erring out.